app/jenkins-for-jira-ui/src/components/CopiedToClipboard/CopiedToClipboard.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import EditorSuccessIcon from '@atlaskit/icon/glyph/editor/success';
+import { copyToClipboard, copyToClipboardContainer } from '../ServerManagement/ServerManagement.styles';
+
+const CopiedToClipboard = () => {
+	return (
+		<div className={cx(copyToClipboardContainer)}>
+			<EditorSuccessIcon
+				primaryColor="#23a06b"
```

Comment:
/nit - is it worth the effort to do something like... primaryColor="var(--success-color)"?

Reply:
Good call out. Let me create a ticket - styling is a bit of a mess right now as we have the initial implementation from when we first built the app, and then what we switched to half way through that process. A lot of it will get ripped out once this new flow has completely rolled out. I then want to go back and tidy up the rest. Throwing some CSS variables in for commonly used styling would be killer.